pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
```

Comment:
what is the origin of this file?

Reply:
Almost the same as [UCX](https://github.com/openucx/ucx/blob/master/bindings/java/checkstyle.xml) one

Reply:
Using scalastyle from spark repo at build time and java checkstyle from SparkRDMA - since in spark repo there's other file dependencies.